OracleIdentityGovernance/dockerfiles/12.2.1.4.0/container-scripts/set_feurl.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
```

Comment:
Oracle Linux doesn't support unversioned Python, so this is unlikely to work. If you don't invoke this scrpt as an executable and only call it via the `python3` interpreter, you can/should remove this line. If you do want to call it like an executable, it needs to refer to `python3`

Reply:
Have fixed both linter issue comment and python comment.